### ISSUE TITLE
Full Lava Tile

### ISSUE THREAD

**Issue by datahead8888** [Secretchronicles/TSC#684]
It would be good to have a lava tile that occupies an entire tile.
These could then be stacked and then combined with the current lava tile to create deep lava pits.

It might be possible to make these passive if the player never can touch them, but we could allow it to be either massive or passive to be safe.

**Comment by Quintus** [comment]
Am 09. Mai 2020 um 12:16 Uhr -0700 schrieb Chris Jacobsen:
>It would be good to have a lava tile that occupies an entire tile.

What is an "entire tile"? The lava element is actually pretty large 
already, compared to other tiles (e.g., sand). Do you mean it should be 
as large as an entire screen?

  -quintus

-- 
Blog: https://mg.guelker.eu

**Comment by datahead8888** [comment]
Yes, you are correct -- I mistakenly assumed a uniform tile size without thinking.

What I meant is that there should be a lava tile that is a full rectangle.  The existing lava tiles do not work as well for stacking, since they a bit short and not straight across toward the top for the graphic.
